src/examples/paraview-vis/paraview_ascent_source.py

```diff
@@ -435,39 +435,41 @@ def RequestData(self, request, inVector, outVector):
     def RequestInformation(self, request, inVector, outVector):
         for outputPort in range(self.GetNumberOfOutputPorts()):
             oi = outVector.GetInformationObject(outputPort)
-            if (self._outputType != vtkConstants.VTK_UNSTRUCTURED_GRID):
+            _topology = self._topologies[outputPort]
+            _coords = self._coords[outputPort]
```

Comment:
I got rid of the `self.` because I redefined those. Maybe I should call them something else for it to be clearer?

Reply:
Do you refer to _topology? That does seem to be a field so it should be prefixed with self. for consistency isn't it?

Reply:
It's just a utility variable to avoid repeating `self._coords[outputPort]` we don't want to store it in the object, so no `self`.